static/app/stores/groupStore.tsx

```diff
@@ -18,30 +18,12 @@ function showAlert(msg, type) {
   IndicatorStore.addMessage(msg, type, {duration: 4000});
 }
 
-// TODO(ts) Type this any better.
-type Change = [string, string, any];
+type ChangeId = string;
 
-class PendingChangeQueue {
-  changes: Change[] = [];
-
-  getForItem(itemId: string) {
-    return this.changes.filter((change: Change) => change[1] === itemId);
-  }
-
-  push(changeId: string, itemId: string, data: any) {
-    this.changes.push([changeId, itemId, data]);
-  }
-
-  remove(changeId: string, itemId?: string) {
-    this.changes = this.changes.filter(
-      change => change[0] !== changeId || change[1] !== itemId
-    );
-  }
-
-  forEach(...args: any[]) {
-    this.changes.forEach.apply(this.changes, args);
-  }
-}
+type Change = {
+  data: any;
```

Comment:
`// TODO(ts) Type this any better.` was for the `any` here. But a bunch of other 'changes' are typed as any too so 🤷